### PR TITLE
Dropdown of multiple map services on map toolbar

### PR DESCRIPTION
**Pull request by smileytechguy**
Added a Pokévision button to map, instead of my old PR with it as a separate window.  Also, made the autofollow and GPX buttons look a lot better

### CODE REVIEW

**Comment by kasagon** [comment]
Isn't Pokevision offline at the moment?

**Comment by smileytechguy** [comment]
It is, but hopefully it'll be back soon

**Comment by smileytechguy** [comment]
Ill make another commit with more mapping services

**Comment by smileytechguy** [comment]
Now contains a dropdown listing multiple map services on the toolbar, including Google Maps, Bing Maps, ,OSM, and coordinates

File: src/main/resources/ink/abb/pogo/scraper/gui/index.html [hunk 1]
```diff
@@ -97,22 +97,6 @@
             height: 400px;
         }
 
-        #pokemon_list {
```
Comment (langerhans) [inline src/main/resources/ink/abb/pogo/scraper/gui/index.html]: Removing that breaks the design of the pokebank

File: src/main/resources/ink/abb/pogo/scraper/gui/index.html [hunk 2]
```diff
@@ -338,10 +439,22 @@
...
+                </div>
+                <div class="map-button">
+                    <h4>Autofollow bot</h4>
+                    <div style="display: block" class="autofollow-slide">
+                        <input type="checkbox" value="None" id="autofollow-slide" name="check" checked />
+                        <label for="autofollow-slide"></label>
+                    </div>
+                </div>
+                <div class="map-button">
+                    <div class="btn" id="pokevision-button">Pokévision</div>
```
Comment (langerhans) [inline src/main/resources/ink/abb/pogo/scraper/gui/index.html]: Well Pokevision is down? Will they come back?

**Comment by langerhans** [comment]
The slider doesn't seem to work for me. The map still follows. I also don't see the mentioned dropdown :open_mouth:

File: src/main/resources/ink/abb/pogo/scraper/gui/index.html (hunk 2 above)
Comment (smileytechguy) [inline src/main/resources/ink/abb/pogo/scraper/gui/index.html]: We're not sure, they said they are abiding by Niantic's wishes and hope that the rules will change in the future.  Check out their twitter twitter.com/PokeVisionGo

File: src/main/resources/ink/abb/pogo/scraper/gui/index.html (hunk 1 above)
Comment (smileytechguy) [inline src/main/resources/ink/abb/pogo/scraper/gui/index.html]: oops, didnt mean to remove that, thought I had put it bace

**Comment by smileytechguy** [comment]
@langerhans I think a commit was left out or broke.  One sec

**Comment by smileytechguy** [comment]
welp, it looks like I overwrote my changes

**Comment by smileytechguy** [comment]
ill be closing for now

**Pull request closed without merging**